registrations/models.py

```diff
@@ -345,6 +350,30 @@ def signups_count(self):
     def attending_signups(self):
         return self.signups.filter(attendee_status=SignUp.AttendeeStatus.ATTENDING)
 
+    @transaction.atomic
+    def anonymize(self):
+        faker = Faker("fi_FI")
+
+        # Allow to anonymize signup only once
+        if self.anonymization_time is None:
+            if contact_person := getattr(self, "contact_person", None):
+                contact_person.anonymize()
+
+            if protected_data := getattr(self, "protected_data", None):
+                protected_data.extra_info = faker.paragraph(nb_sentences=1)
+                protected_data.save()
```

Comment:
Hmm, is there a reason why protected data doesn't have `anonymize()` method like other models that can be anonymized? If not, I would like it to be added for consistency reasons.